src/AddTask.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FaPlus, FaEdit } from 'react-icons/fa';
+import './App.css';
+
+const AddTask = (props) => {
+  const { input, setInput, handleAddTask, editIndex } = props;
+
+  return (
+    <div>
+      <div className="input-group mb-3">
+        <input
+          type="text"
+          className="form-control"
+          placeholder="Enter a task..."
+          value={input}
+          onChange={(e) => setInput(e.target.value)}
+        />
+        <button className="btn btn-primary" onClick={handleAddTask}>
+          {editIndex !== null ? <FaEdit /> : <FaPlus />}
```

Comment:
This is not necessary, you can simply do `{!!editIndex ? <FaEdit /> : <FaPlus />}`

Reply:
@ErickMwazonga I have addressed your comment; please review again